### PR TITLE
Moved the JSON parsing and DB processing to an endpoint

### PR DESCRIPTION
**Pull request by SajuJ**
Moved the JSON parsing and DB processing from index.js to an endpoint

### CODE REVIEW

File: src/helpers/db/populate-db.js [hunk 1]
```diff
@@ -48,105 +48,118 @@ const collectionPestDistribution = 'PEST_DISTRIBUTION'
...
+            )
+            await loadData(
+              filePathPestPras,
+              mongoUri,
+              server.db,
+              collectionPestPras,
+              1
+            )
+
+            // await server.start()
```
Comment (manabshy) [inline src/helpers/db/populate-db.js:148]: why await server.start() is commented. ?.

Comment (SajuJ) [reply src/helpers/db/populate-db.js:148]: Is it required in your file? There's already one in async in index.js, function startServer() {
  const server = await createServer()
  await server.start()}

Comment (manabshy) [reply src/helpers/db/populate-db.js:148]: Please remove or add back again if everything still works.

Comment (SajuJ) [reply src/helpers/db/populate-db.js:148]: I think the server should start in createServer (this was already there from CDP) and the processing will happen in the background, I have tested it and didnt see any issues.

**Review by manabshy — APPROVED** [review]

**Pull request merged**